example/example-etcd-cluster-with-abs-backup.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: "etcd.database.coreos.com/v1beta2"
```

Comment:
The example in spec_examples.md is good enough.
To keep it consistent, no duplicate example here again.